arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringViewer.vue

```diff
@@ -0,0 +1,6 @@
+<script setup lang="ts">
+const props = defineProps<{ value?: string | null }>();
+</script>
+<template>
+    <p>{{ props.value }}</p>
```

Comment:
should this be a p-element? idk this may warrant larger discussion. What would we expect the read-only element to be in the case of _every_ widget? In my head for this one it's a disabled input, but I do not feel strongly about this. @robgaston ?

Reply:
IMO a non-form element will look better in a report, but have no particular preference - I'll defer to consensus.